src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(y) {
+    count++;
+
+    return function(x) {
+      return count === 4 && x + y % 2 !== 0
```

Comment:
add brackets to understand what happens here more clearly

Reply:
Not sure I understand u clearly, but fixed)